### PR TITLE
cleanup: add scope function in CardTemplateBrowserAppearanceEditor

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
cleanup: add scope function in CardTemplateBrowserAppearanceEditor
fix: IDE lint

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt [hunk 1]
```diff
@@ -159,13 +159,15 @@ class CardTemplateBrowserAppearanceEditor : AnkiActivity() {
     private fun saveAndExit() {
         Timber.i("Save and Exit")
         val data = Intent()
-        data.putExtra(INTENT_QUESTION_FORMAT, questionFormat)
-        data.putExtra(INTENT_ANSWER_FORMAT, answerFormat)
+        data.apply {
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt:162]: Why not `Intent().apply {`

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt [hunk 2]
```diff
@@ -217,9 +219,10 @@ class CardTemplateBrowserAppearanceEditor : AnkiActivity() {
...
         @CheckResult
         fun getIntent(context: Context, questionFormat: String, answerFormat: String): Intent {
-            val intent = Intent(context, CardTemplateBrowserAppearanceEditor::class.java)
-            intent.putExtra(INTENT_QUESTION_FORMAT, questionFormat)
-            intent.putExtra(INTENT_ANSWER_FORMAT, answerFormat)
+            val intent = Intent(context, CardTemplateBrowserAppearanceEditor::class.java).apply {
+                putExtra(INTENT_QUESTION_FORMAT, questionFormat)
+                putExtra(INTENT_ANSWER_FORMAT, answerFormat)
+            }
             return intent
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt:226]:
```suggestion
            return Intent(context, CardTemplateBrowserAppearanceEditor::class.java).apply {
                putExtra(INTENT_QUESTION_FORMAT, questionFormat)
                putExtra(INTENT_ANSWER_FORMAT, answerFormat)
            }
```

**Review by david-allison — CHANGES_REQUESTED** [review]
Loosks better, minor suggestions

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt (hunk 1 above)
Comment (oyeraghib) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt]: fixed that!

**Review by lukstbit — APPROVED** [review]

**Comment by lukstbit** [comment]
@oyeraghib Please apply David's suggestion from above.

**Comment by oyeraghib** [comment]
> @oyeraghib Please apply David's suggestion from above.

sorry for being late. Just updated.

**Review by lukstbit — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
Verified David's suggestions are implemented, and in general looks to good to me - thanks!

**Pull request merged**